task_02/index.html

```diff
@@ -1,11 +1,195 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Document</title>
-</head>
-<body>
-    Task 2
-</body>
-</html>
\ No newline at end of file
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="wrapper">
+      <!-- Desktop navigation -->
+      <nav role="navigation" class="desktop-navigation">
+        <div class="nav-wrapper">
+          <div class="list-item"><a href="" class="links">Home</a></div>
+          <div class="list-item"><a href="" class="links">Approach</a></div>
+          <div class="list-item"><a href="" class="links">Work</a></div>
+          <div class="list-item"><a href="" class="links">People</a></div>
+          <div class="list-item">
+            <a href="" class="links" id="disabled">Jobs</a>
+          </div>
+          <div class="list-item">
+            <a href="" class="links" id="disabled">Blog</a>
+          </div>
+          <div class="list-item">
+            <a href="" class="links" id="disabled">Contact</a>
```

Comment:
I know this is just an example, but keep in mind that ID should be unique.

Reply:
This disabled can also be class,since it has just css effect of disabled item.